en/patterns/lists.md

```diff
@@ -1,32 +1,49 @@
 ---
 title: Lists - Design System Pattern
-_description: The Lists Pattern Symbols depict varios scenarios that use lists to organize some common collections of information.
+_description: The Lists Pattern Symbols depict varios scenarios that use lists to organize some common collections of information on one or more levels in horizontal and vertical layout.
 _keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
 ---
 
 ## Lists
 
-> [!Note]
-> With Indigo.Design UI Kits version 1.1.0 the patterns under Lists are no longer available due to an issue in Sketch 52. The issue is related to the many changes Sketch 52 brings along, which ultimately causes the Sketch app to crash. Once the Sketch team resolves this issue, we will push an update to the patterns library including the lists that were available before. In the mean time you can use the List Component and template and style any type of list that you would like as the Components have been fully tested and work fine.
-
-Use the Lists Pattern Symbols as presets for organized collections of data depicting common scenarios such as filterable settings, people, song playlists, searchable lists, and settings.
+Use the Lists Pattern Symbols as presets for organized collections of data in horizontal or vertical layout depicting common scenarios such as people, social media stories, multilevel list elements, settings, recently used items, products, categories of items.
```

Comment:
This is an old issue but can you change multilevel to multiline because we mean multiple label and lines of text :)

Reply:
also add the note that illustrations nested inside the Pattern Symbols cannot be code generated like you have it in the onboarding topic